test/jdk/com/sun/java/swing/plaf/gtk/TestFileChooserSingleDirectorySelection.java

```diff
@@ -191,29 +189,30 @@ private static void setLookAndFeel(UIManager.LookAndFeelInfo laf) {
     private static void doTesting(UIManager.LookAndFeelInfo laf, int xOffset)
                                     throws Exception {
         SwingUtilities.invokeAndWait(() -> {
-            frameLocation = fileChooser.getLocationOnScreen();
-            frameWidth = frame.getWidth();
-            frameHeight = frame.getHeight();
+            Point fileChooserLocation = fileChooser.getLocationOnScreen();
+            fileChooserLocation.y += frame.getHeight() / 3;
+            clickLocation = new Point(fileChooserLocation);
         });
-        clickMouse(frameLocation, 0, frameHeight, xOffset);
+        clickMouse(clickLocation, xOffset);
         checkResult(laf);
     }
 
-    private static void clickMouse(Point point, int width, int height,
-                                   int xOffset) {
-        robot.mouseMove(point.x + width/2 + xOffset , point.y + height/3);
+    private static void clickMouse(Point point, int xOffset) {
+        robot.mouseMove(point.x + xOffset , point.y);
         robot.mousePress(InputEvent.BUTTON1_DOWN_MASK);
         robot.mouseRelease(InputEvent.BUTTON1_DOWN_MASK);
         robot.delay(100);
         robot.waitForIdle();
     }
 
-    private static void checkResult(UIManager.LookAndFeelInfo laf) {
-        File files[] = fileChooser.getSelectedFiles();
-        if (files.length == 0) {
-            throw new RuntimeException("getSelectedFiles returned " +
-                    "empty array for LAF: " + laf.getClassName());
-        }
+    private static void checkResult(UIManager.LookAndFeelInfo laf) throws Exception {
+        SwingUtilities.invokeAndWait(() -> {
+            File files[] = fileChooser.getSelectedFiles();
```

Comment:
```suggestion
            File[] files = fileChooser.getSelectedFiles();
```
Let's use Java-style array declaration.